.github/workflows/e2e-test.yml

```diff
@@ -44,10 +44,10 @@ jobs:
         exclude:
```

Comment:
I can not comment on line 39 so leaving the comment here. I think this line 39 should be updated too? So it should be: 

```
        woocommerce:   [ '5.4.0', '6.3.1', '6.5.1', 'beta' ]
```

Reply:
Fixed.